src/pages/AddNewCake.css

```diff
@@ -0,0 +1,19 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background: var(--color-background);
+}
+
+.btn-submit {
+  background-color: var(--color-text);
+  border: 0px solid white;
```

Comment:
I think what you want to achieve here is that the default border isn't shown.
You could also type this `border: none;`